esp-wifi/MIGRATING-0.9.md

```diff
@@ -39,3 +37,26 @@ You no longer have to set up clocks and pass them to `esp_wifi::initialize`.
      // ...
  }
 ```
+
+## Memory allocation
+
+You now need to have a global allocator provided by `esp-alloc` providing allocations from internal memory
+
+```diff
+ #![no_std]
+ #![no_main]
+
++extern crate alloc;
++
+ use embedded_io::*;
++use esp_alloc as _;
+ use esp_backtrace as _;
+ // ...
+
+ #[entry]
+ fn main() -> ! {
++    esp_alloc::heap_allocator!(72 * 1024);
```

Comment:
🤔 ... why 72?

Reply:
Before the default was around 80k - with 72k I found the examples to still work reliable enough. You can also go a bit lower but you will see a couple of warnings regarding failed allocations

This certainly changes if you use the same heap for your own allocations - probably good to mention this in the migration-guide